rke/k8s_version_info.go

```diff
@@ -614,6 +614,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.6.4-patch0",
 			MinRKEVersion:     "1.3.8-rc0",
 		},
+		"v1.24": {
+			MinRancherVersion: "2.6.6-rc0",
```

Comment:
Let's change this to `patch0`, this is to handle scenarios where we need to tag a custom patch later on for an issue and v1.24 won't load there. We ran into this with v1.23 and had to later update it to `2.6.4-patch0`